### DISCUSSION TITLE
Improve accessibility of inline links

### DISCUSSION THREAD

**Discussion by sofietoft** [GitbookIO/community#233] [Feature Requests]
### What's your feature/idea?

Links in the docs site only use a color variation to indicate they are links.
This can make them difficult if not impossible for some people to see.
Example from our docs (docs.umbraco.com) in dark mode.

![image](https://github.com/GitbookIO/community/assets/24529086/326027e9-2b5c-40c5-b574-f97ec4b7dd51)

I suggest that the link style is updated to use the interaction state style by default, with the interaction state removing the underline - see the example below (again from our docs in dark mode):

![image](https://github.com/GitbookIO/community/assets/24529086/5cfa122a-73df-4fef-9788-4610af90db02) dark mode.

### What's your desired outcome?

The desired outcome would be inline-links that are more visible and accessible to everyone, no matter which colors are chosen for the _theme_.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by widal001** [comment]
This is also a need for our potential use case -- We'd like to use GitBook for a Federal open source project, but the current styling for links (only underline on hover) don't meet the 508 compliance standards of our Federal agency's office. 

Even if link styling did not include an underline by default, it would be great if we could manually add an underline -- I tried to do this by using `cmd + U` and it did add an underline but the underline didn't remain after I refreshed the page.
